src/app/components/client/exposure_card/ScanResultCard.tsx

```diff
@@ -202,7 +203,10 @@ export const ScanResultCard = (props: ScanResultCardProps) => {
       <div className={styles.exposureCard}>
         <div className={styles.exposureHeader}>
           <dl className={styles.exposureHeaderList}>
-            <dt className={styles.visuallyHidden}>
+            <dt
+              className={styles.visuallyHidden}
+              aria-hidden={props.isWideScreen ? false : true}
```

Comment:
Since `props.isWideScreen` is already Boolean, does this need the ternery, or could we do something like:
```suggestion
              aria-hidden={!props.isWideScreen}
```

Not sure if that is better in any way, but it is shorter by about 14 chars. 🤷